yarn.lock

```diff
@@ -0,0 +1,5490 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
This project is using npm instead of yarn. I think we should not keep both package-lock.json and yarn.lock.

Considering the scope of this change, it's better to remove this yarn.lock and keep using npm for now.